scripts/bert/utils.py

```diff
@@ -0,0 +1,96 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Utility functions for BERT."""
+import json
+import hashlib
+import gluonnlp
+from tensorflow.python import pywrap_tensorflow
```

Comment:
This forces the users to install tf in order to run all the Bert experiments in Gluon.

Reply:
This script is currently only used by convert.py. Other scripts will not import this file. Maybe I should rename the file to make it clearer.